core/src/components/datetime/datetime.scss

```diff
@@ -267,19 +267,24 @@ ion-picker-column-internal {
   justify-content: space-between;
 }
 
-:host .calendar-action-buttons ion-item,
+:host .calendar-action-buttons .calendar-month-year-toggle {
```

Comment:
Does this rule need to be so specific now, or can we remove the `:host .calendar-action-buttons` piece? Ditto for the `ion-label` and `ion-icon` styles below. We may also be able to fold this first one into the `.calendar-month-year-toggle` rule near the bottom of this file.